app/src/main/java/com/example/tour_a_versity_v3/CechPhotos.kt

```diff
@@ -4,7 +4,7 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.example.tour_a_versity_v3.R
 
-class CECH_Photos: AppCompatActivity() {
+class CechPhotos: AppCompatActivity() {
```

Comment:
changed the name of this class as no underscore allowed in Kotlin according to naming rules: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules